app/src/main/java/org/sopt/dosopttemplate/presentation/auth/SignUpViewModel.kt

```diff
@@ -0,0 +1,127 @@
+package org.sopt.dosopttemplate.presentation.auth
+
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.Transformations
+import androidx.lifecycle.ViewModel
+import org.sopt.dosopttemplate.server.ServicePool
+import org.sopt.dosopttemplate.server.auth.SignUpReq
+import retrofit2.Call
+import retrofit2.Callback
+import retrofit2.Response
+
+class SignUpViewModel : ViewModel() {
+
+    private val _signupResult = MutableLiveData<Boolean>()
+    val signupResult: LiveData<Boolean> get() = _signupResult
+
+    private val _isIdValid = MutableLiveData<Boolean>()
```

Comment:
```suggestion
private val _isIdValid = MutableLiveData(false)
    val isIdValid: LiveData<Boolean> get() = _isIdValid
```
이렇게 써주면 initializeDefaultValues() 필요없이 기본값을 false로 초기화해줍니다! 다른 liveData 변수도 이런식으로 바꿔주세용